crates/cxx-qt/Cargo.toml

```diff
@@ -19,3 +19,5 @@ categories = ["api-bindings", "gui"]
 
 [dependencies]
 cxx-qt-macro.workspace = true
+cxx.workspace = true
```

Comment:
why was this added, it seems unused ?

Reply:
Ah maybe for the docs tests ?

Then it goes under `[dev-dependencies]` not `[dependencies]`

Reply:
Done